rich-default/environment.yml

```diff
@@ -0,0 +1,522 @@
+  - r-usethis=1.5.1=r36h6115d3f_1
+  - r-utf8=1.1.4=r36hcdcec82_1001
+  - r-uuid=0.1_2=r36hcdcec82_1002
+  - r-vctrs=0.2.1=r36hcdcec82_0
+  - r-viridis=0.5.1=r36h6115d3f_1003
+  - r-viridislite=0.3.0=r36h6115d3f_1002
+  - r-whisker=0.4=r36h6115d3f_0
+  - r-withr=2.1.2=r36h6115d3f_1001
+  - r-xfun=0.11=r36h6115d3f_0
+  - r-xml2=1.2.2=r36h0357c0b_0
+  - r-xopen=1.0.0=r36h6115d3f_1002
+  - r-xtable=1.8_4=r36h6115d3f_2
+  - r-xts=0.11_2=r36hcdcec82_1
+  - r-yaml=2.2.0=r36hcdcec82_1002
+  - r-zeallot=0.1.0=r36h6115d3f_1001
+  - r-zoo=1.8_6=r36hcdcec82_1
+  - rdkit=2017.09.1=py36_1
+  - readline=8.0=hf8c457e_0
+  - requests=2.22.0=py36_1
+  - resonance=0.19.1=py36_1000
+  - rpy2=3.1.0=py36r36hc1659b7_3
+  - ruamel_yaml=0.15.71=py36h14c3975_1000
+  - scikit-image=0.16.2=py36hb3f55d8_0
+  - scikit-learn=0.22=py36hcdab131_1
+  - scipy=1.4.1=py36h921218d_0
+  - scotch=6.0.8=h0eec0ba_1
+  - seaborn=0.9.0=py_2
+  - sed=4.7=h1bed415_1000
+  - send2trash=1.5.0=py_0
+  - setuptools=44.0.0=py36_0
+  - simpervisor=0.3=py_1
+  - simplegeneric=0.8.1=py_1
+  - six=1.13.0=py36_0
+  - sortedcontainers=2.1.0=py_0
+  - soupsieve=1.9.4=py36_0
+  - sqlalchemy=1.3.12=py36h516909a_0
+  - sqlite=3.29.0=hcee41ef_0
+  - statsmodels=0.10.2=py36hc1659b7_0
+  - sympy=1.5.1=py36_0
+  - tbb=2018.0.5=h2d50403_0
+  - tblib=1.6.0=py_0
+  - terminado=0.8.3=py36_0
+  - testpath=0.4.4=py_0
+  - tiledb=1.6.2=h69c774e_1
+  - tini=0.18.0=h14c3975_1001
+  - tk=8.6.9=hed695b0_1002
+  - tktable=2.10=h555a92e_3
+  - toolz=0.10.0=py_0
+  - tornado=6.0.3=py36h516909a_0
+  - tqdm=4.41.1=py_0
+  - traitlets=4.3.3=py36_0
+  - traittypes=0.2.1=py_1
+  - typing_extensions=3.7.4.1=py36_0
+  - tzcode=2019a=h516909a_1002
+  - tzlocal=2.0.0=py_0
+  - udunits2=2.2.27.6=h4e0c4b3_1001
+  - urllib3=1.25.7=py36_0
+  - vincent=0.4.4=py_1
+  - wcwidth=0.1.8=py_0
+  - webencodings=0.5.1=py_1
+  - wheel=0.33.6=py36_0
+  - widgetsnbextension=3.5.1=py36_0
+  - xarray=0.14.1=py_1
+  - xerces-c=3.2.2=h8412b87_1004
+  - xlrd=1.2.0=py_0
+  - xorg-fixesproto=5.0=h14c3975_1002
+  - xorg-inputproto=2.3.2=h14c3975_1002
+  - xorg-kbproto=1.0.7=h14c3975_1002
+  - xorg-libice=1.0.10=h516909a_0
+  - xorg-libsm=1.2.3=h84519dc_1000
+  - xorg-libx11=1.6.9=h516909a_0
+  - xorg-libxau=1.0.9=h14c3975_0
+  - xorg-libxdmcp=1.1.3=h516909a_0
+  - xorg-libxext=1.3.4=h516909a_0
+  - xorg-libxfixes=5.0.3=h516909a_1004
+  - xorg-libxi=1.7.10=h516909a_0
+  - xorg-libxrender=0.9.10=h516909a_1002
+  - xorg-libxt=1.2.0=h516909a_0
+  - xorg-renderproto=0.11.1=h14c3975_1002
+  - xorg-xextproto=7.3.0=h14c3975_1002
+  - xorg-xproto=7.0.31=h14c3975_1007
+  - xz=5.2.4=h14c3975_1001
+  - yaml=0.1.7=h14c3975_1001
+  - yarl=1.3.0=py36h516909a_1000
+  - zeromq=4.3.2=he1b5a44_2
+  - zict=1.0.0=py_0
+  - zipp=0.6.0=py_0
+  - zlib=1.2.11=h516909a_1005
+  - zstd=1.4.0=h3b9ef0a_0
+  - pip:
+    - biopandas==0.2.5
+    - ipymol==0.5
+    - pypdb==1.300
+    - pyqt5-sip==4.19.18
+    - pyqtwebengine==5.12.1
+    - xmltodict==0.12.0
```

Comment:
Ideally we create conda-forge recipes for these, so we can fully manage with conda.

Reply:
@moorepants, is it usually the case to ask the maintainers whether they are okay with submitting to conda-forge first, before doing it ourselves?
`pypdb` is already on conda-forge but with an older version. The new version is only available on pip. I would rather submit the PR (after I make a few changes) to use `environment.yml` and later submit an issue regarding conda-forge recipes.